torchaudio/functional/functional.py

```diff
@@ -1631,3 +1632,38 @@ def rnnt_loss(
         return costs.sum()
 
     return costs
+
+
+def compute_power_spectral_density_matrix(
+    specgram: Tensor,
+    mask: Optional[Tensor] = None,
+    normalize: bool = True,
+    eps: float = 1e-10,
+) -> Tensor:
+    """Compute cross-channel power spectral density (PSD) matrix.
+    Args:
+        specgram (Tensor): Multi-channel complex-valued spectrum.
+            Tensor of dimension `(..., channel, freq, time)`
+        mask (Tensor or None, optional): Real-valued Time-Frequency mask
+            for normalization. Tensor of dimension `(..., freq, time)`
+            (Default: ``None``)
+        normalize (bool, optional): whether normalize the mask along the time dimension.
```

Comment:
```suggestion
        normalize (bool, optional): whether to normalize the mask along the time dimension.
```

also, default arg

and a couple more:
- our spectrogram functions accept parameter "normalized" — should we adjust here for consistency's sake?
- we already expect users to construct masks outside the function, and normalization doesn't depend on anything else inside the function. wondering how reasonable it'd be to remove `normalize` entirely

Reply:
That's a good question. Here `normalize` is intended to normalize the `psd` by the sum of the mask values on the time dimension.

Suppose we have an estimated mask by some algorithm, users may want to compute the psd matrix without modifying the mask, like normalizing it manually.

spectrogram has to use `normalized` because `torch.stft` requires such name. In `torch.load` we also have `normalize` argument, so I think it's fine.